Public/javascripts/search_core.js

```diff
@@ -200,7 +200,7 @@ export class SPISearchCore {
 
     // Repository identifier.
     const repositoryElement = document.createElement('small')
-    repositoryElement.textContent = result.owner + '/' + result.package_name
+    repositoryElement.textContent = result.owner + '/' + result.packageName
```

Comment:
Is this sufficient to regen the minified versions?

Reply:
Yea, I'll commit a new minified version on this branch now.